db/migrate/20150729044245_add_name_designation_to_user.rb

```diff
@@ -0,0 +1,8 @@
+class AddNameDesignationToUser < ActiveRecord::Migration
+  def change
+    change_table :users do |t|
+      t.string :name
+      t.string :designation
```

Comment:
For values like Mr, Mrs, Ms, Dr etc

On Fri, Jul 31, 2015 at 4:36 PM, Jaspreet Singh <notifications@github.com>
wrote:

> In db/migrate/20150729044245_add_name_designation_to_user.rb
> <https://github.com/SAkshat/iconf/pull/3#discussion_r35964706>:
>
> > @@ -0,0 +1,8 @@
> > +class AddNameDesignationToUser < ActiveRecord::Migration
> > +  def change
> > +    change_table :users do |t|
> > +      t.string :name
> > +      t.string :designation
>
> what is designation column for?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SAkshat/iconf/pull/3/files#r35964706>.
>



-- 

Akshat Singhal
Vinsol